org-formation/090-systems-manager/QuickSetup-PatchPolicy-Prod.yaml

```diff
@@ -0,0 +1,56 @@
+AWSTemplateFormatVersion: '2010-09-09'
```

Comment:
what is the significance of `Prod` in the filename?  do you ever plan to have a `dev` version?

Reply:
Oops, that's a remnant of when I was only going to apply this to prod accounts